### PR TITLE
View broker skew in kafka-ui

### PR DESCRIPTION
**Pull request by NeiruBugz**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
- add new columns to brokers table
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/common/NewTable/SizeCell.tsx [hunk 1]
```diff
@@ -3,8 +3,15 @@ import { CellContext } from '@tanstack/react-table';
 import BytesFormatted from 'components/common/BytesFormatted/BytesFormatted';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const SizeCell: React.FC<CellContext<any, unknown>> = ({ getValue }) => (
-  <BytesFormatted value={getValue<string | number>()} />
+type AsAny = any;
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/common/NewTable/SizeCell.tsx:6]: why are we added this ?

Comment (NeiruBugz) [reply kafka-ui-react-app/src/components/common/NewTable/SizeCell.tsx:6]: After changes of the `SizeCell` props the props string needs to be formatted as below and the eslint-disable comment ignored, so I added `AsAny`, to ignore it before props declaration

File: kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx [hunk 2]
```diff
@@ -98,10 +104,47 @@ const BrokersList: React.FC = () => {
...
+          />
+        ),
+      },
+      { header: 'Leaders', accessorKey: 'partitionsLeader' },
+      {
+        header: 'Leader skew',
+        accessorKey: 'leadersSkew',
+        // eslint-disable-next-line react/no-unstable-nested-components
+        cell: ({ getValue }) => (
+          <ColoredCell
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx:130]: lets not call `getValue` multiple times lets just call it once and put it in a variable

Comment (NeiruBugz) [reply kafka-ui-react-app/src/components/Brokers/BrokersList/BrokersList.tsx]: Updated

**Review by Mgrdich — APPROVED** [review]

**Review by Haarolean — APPROVED** [review]

**Pull request merged**